src/pivot-table/hooks/use-items-rendered-handler/utils/get-row-pages.ts

```diff
@@ -36,8 +36,8 @@ const getRowPages = (
       const page = {
         qLeft,
         qTop: Math.max(0, top + pageInfo.page * pageInfo.rowsPerPage),
-        qHeight: 1,
         qWidth,
+        qHeight: 1,
```

Comment:
any reason for this change?

Reply:
Yes hovering the return values of `getRowPages` in VCS the output type would be `{ qLeft, qTop, qHeight, qWidth }`. It's now changed to `{ qLeft, qTop, qWidth, qHeight }` which is the "correct" property order.